Sources/FluentKit/Properties/Children.swift

```diff
@@ -1,125 +1,191 @@
 @propertyWrapper
-public final class Children<From, To>: AnyProperty, AnyEagerLoadable
-    where From: Model, To: Model
+public final class Children<F, T>: AnyProperty, AnyEagerLoadable
+    where F: ModelIdentifiable, T: Model
```

Comment:
Do we need `F: ModelIdentifiable` here? It might be cleaner to have the `idValue` storage be an `Any` type that is get/set as a certain type in our typed-extensions. Like this maybe:

```swift
final class Children<From, To> {
    var anyIDValue: Any?
    ...
}

extension Children where To: Model {
    var idValue: To.IDValue? { 
        get { ... }
        set { ... }
    }
}

extension Children where To: OptionalType, To.Wrapped: Model {
    var idValue: To.Wapped.IDValue? { 
        get { ... }
        set { ... }
    }
}
```